app/javascript/app/components/button/share-button/share-button-styles.scss

```diff
@@ -0,0 +1,99 @@
+@import '~styles/layout.scss';
+
+.shareButton {
+  width: 118px;
+
+  .shareText {
+    margin-left: 10px;
+  }
+
+  &Active {
+    fill: $theme-color;
+  }
+
+  &:hover {
+    box-shadow: none;
+    fill: $theme-color;
+  }
+}
+
+.modal {
+  height: initial;
+
```

Comment:
Maybe we can set `width: 90%` or `margin: 0 20px` for mobiles? Because now it has 100% width:
![Screenshot from 2019-12-27 10-55-53](https://user-images.githubusercontent.com/15097138/71514634-84b74f00-2897-11ea-81a4-c7b1ac8ec37b.png)